storage/reads/array_cursor.gen.go.tmpl

```diff
@@ -0,0 +1,581 @@
+package reads
```

Comment:
```
$ git diff master -- storage/reads/array_cursor.gen.go.tmpl
diff --git a/storage/reads/array_cursor.gen.go.tmpl b/storage/reads/array_cursor.gen.go.tmpl
index a1c3f27888..428310dec7 100644
--- a/storage/reads/array_cursor.gen.go.tmpl
+++ b/storage/reads/array_cursor.gen.go.tmpl
@@ -7,8 +7,8 @@ import (

     "github.com/influxdata/flux/interval"
     "github.com/influxdata/flux/values"
-    errors2 "github.com/influxdata/influxdb/v2/kit/platform/errors"
-       "github.com/influxdata/influxdb/v2/tsdb/cursors"
+       errors2 "github.com/influxdata/influxdb/kit/platform/errors"
+       "github.com/influxdata/influxdb/tsdb/cursors"
 )

 const (
```

Reply:
It looks like this file is new - did we copy the .gen.go without copying the .gen.go.tmpl before?

Reply:
Yes (at least as far as I can tell from the git history)